### ISSUE TITLE
Everything is not detected

### ISSUE THREAD

**Issue by gomico** [dnGrep/dnGrep#1188]
Installing Everything 1.5a seems to break the integration.
I used to have the Everything option in pattern type, next to regex and wildcard. Installed Everything 1.5a today and found that the option in dnGrep disappears.
The original Everything 1.4 still exists in the system, but dnGrep fails to detect it.
Is there any way to fix the integration, maybe using some environment variable?

**Comment by doug24** [comment]
I have not installed Everything 1.5a on any computer.  dnGrep uses the Everything SDK, and it is this SDK that locates the Everything installation, not any dnGrep code.  It appears the 1.4 SDK can't find the 1.4 installation anymore.

I will try installing it on another computer to see if I can see any errors.

**Comment by doug24** [comment]
On another computer, I updated to the latest Everything 1.4.1 release, verified dnGrep is working with Everything, then installed the latest 1.5a release.  When I started dnGrep, it loaded the existing Everything SDK and said it found the 1.4.1 release, and the Everything search option appears in dnGrep.  So, from what I see, it is working correctly.

I don't know what to suggest other than uninstalling Everything 1.4 and reinstall the latest 1.4.
